modules/indexExchangeBidAdapter.js

```diff
@@ -419,6 +426,9 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
     _IndexRequestData.targetAggregate = {'open': {}, 'private': {}};
 
     if (!utils.hasValidBidRequest(bidArr[0].params, requiredParams, ADAPTER_NAME)) {
+      for (var i = 0; i < bidArr.length; i++) {
+        passOnBid(bidArr[i].placementCode);
+      }
```

Comment:
@dugwood Referencing: https://github.com/prebid/Prebid.js/blob/master/src/bidmanager.js#L54
`bidmanager` is expecting the number of bids returned from Index Exchange to match with the number of sizes in the slots.